docs/technical-design/database-diagram.md

```diff
@@ -0,0 +1,110 @@
+# Diagram of the db with future designs
+
+## Current DB
+
+```mermaid
+erDiagram
+
+HealthPlanPackageTable ||--|{ HealthPlanRevisionTable: "has many"
+HealthPlanPackageTable }|--|| State: "has many"
+
+HealthPlanPackageTable {
+    string id
+    string stateCode
+}
+
+HealthPlanRevisionTable {
+    string id
+    string pkgID
+    datetime createdAt
+    bytes   formDataProto
+    datetime submittedAt
+    string submittedBy
+    string submittedReason
+    datetime unlockedAt
+    string unlockedBy
+    string unlockedReason
+}
+
+State {
+    string stateCode
+    string name
+    int latestSubmissionNumber
+}
+
+User {
+    int id
+    string givenName
+    string familyName
+    string email
+    string role
+}
+
+User }|--|{ State: "many to many"
+```
+
+## QA Doc Upload Design
+
+```mermaid
+
+erDiagram
+
+HealthPlanPackageTable ||--|{ HealthPlanRevisionTable: "has many"
+HealthPlanPackageTable }|--|| State: "has many"
+
+HealthPlanPackageTable {
+    string id
+    string stateCode
+}
+
+HealthPlanRevisionTable {
+    string id
+    string pkgID
+    datetime createdAt
+    bytes   formDataProto
+    datetime submittedAt
+    string submittedBy
+    string submittedReason
+    datetime unlockedAt
+    string unlockedBy
+    string unlockedReason
+}
+
+State {
+    string stateCode
+    string name
+    int latestSubmissionNumber
+}
+
+User {
+    int id
+    string givenName
+    string familyName
+    string email
+    string role
+}
+
+User }|--|{ State: "many to many"
+
+QuestionRound {
+    string id
```

Comment:
Maybe its implicit here but `createdAt` is missing. Assuming we would build in a way that we could have drafts